package.json

```diff
@@ -14,9 +14,9 @@
     "test:unit": "vue-cli-service test:unit"
   },
   "dependencies": {
-    "apollo-boost": "^0.4.3",
+    "apollo-boost": "^0.4.0",
```

Comment:
Downgrade?

Reply:
Oh, interesting, good spot.

When I run `ncu -u`, it displays normal colors for most updates. But when it goes above the range set in `packages.json`, then it displays the upgrade in red.

I noticed `vuetify` had a major release, so removed it. And accidentally reset `apollo-boost` back to 0.4.0. It was not changed in the lock file (`package-lock.json`) as the current version passes the check.

But it should instead go up to 0.4.4. Will update it later. Thanks!!

Reply:
Fixed, thanks!